lib/WP_Auth0_LoginManager.php

```diff
@@ -283,25 +287,13 @@ public function redirect_login() {
 			$userinfo_resp_body = wp_remote_retrieve_body( $userinfo_resp );
 		}
 
-		// Management API call failed, fallback to userinfo.
-		if ( 200 !== $userinfo_resp_code || empty( $userinfo_resp_body ) ) {
-
-			$userinfo_resp      = WP_Auth0_Api_Client::get_user_info( $auth_domain, $access_token );
-			$userinfo_resp_code = (int) wp_remote_retrieve_response_code( $userinfo_resp );
-			$userinfo_resp_body = wp_remote_retrieve_body( $userinfo_resp );
-
-			if ( 200 !== $userinfo_resp_code || empty( $userinfo_resp_body ) ) {
-
-				WP_Auth0_ErrorManager::insert_auth0_error( __METHOD__ . ' L:' . __LINE__, $userinfo_resp );
-				throw new WP_Auth0_LoginFlowValidationException(
-					__( 'Error getting user information', 'wp-auth0' ),
-					$userinfo_resp_code
-				);
-			}
+		// Management API call failed, fallback to ID token.
```

Comment:
What would be the expectation for a management API call to fail?

Reply:
If Client Credential Grant is turned off